### ISSUE TITLE
Allow setting only a subdomain to run in the container, not the entire site

### ISSUE THREAD

**Issue by zerodogg** [mozilla/contain-facebook#685]
My use-case for this is that I'm testing FB integration in an app in a browser with FB container enabled. I'd like to allow just the subdomain with the FB integration to run in the FB container, while the toplevel domain runs in another (multi-account containers-defined) container.

**Comment by maxxcrawford** [comment]
Got'cha. I would recommend using the [Multi-Account Containers](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/) instead of Facebook Container. It allows for more granular control of what sites are in which containers, to a single (sub) domain granularity.

**Comment by zerodogg** [comment]
I've actually got it installed as well, and I'm using both. Facebook Container for convenience since it nicely wraps up fb, while multi-account does everything else. And I like knowing it blocks fb outright by default. However, for this one site, it has a FB widget embedded, and with fb containers enabled it won't display it. The site itself is in another multi acc container, but fb container still blocks the embed - which is usually what I want, but not in this single instance. I did end up just whitelisting the entire domain, but it's a sledgehammer approach.

**Comment by maxxcrawford** [comment]
Agreed. We made Facebook Container to be the more "entry-level" of the Containers extensions, which is why we made the site-adding feature to be more broad. We're exploring a more expansive preferences panel. We'll be sure to include this issue in those discussions. 

Also note, we're considering adding the same functionality/option to MAC, so that users can set containers preferences at a entire domain level.
